### PR TITLE
Add Cas-OFFinder

### PR DESCRIPTION
**Pull request by pjb7687**
Homepage: https://github.com/snugel/cas-offinder

### CODE REVIEW

**Comment by Biswa96** [comment]
Conflicts with libproxy??

**Comment by pjb7687** [comment]
> Conflicts with libproxy??

File: mingw-w64-cas-offinder/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    -DCMAKE_BUILD_TYPE=Release \
+    ../${_realname}-${pkgver}
+
+  make
+}
+
+package() {
+  cd ${srcdir}/${_realname}-${pkgver}
+  mkdir -p ${pkgdir}${MINGW_PREFIX}/bin/
+  cp ${srcdir}/build-${MINGW_CHOST}/cas-offinder ${pkgdir}${MINGW_PREFIX}/bin/
```
Comment (lazka) [inline mingw-w64-cas-offinder/PKGBUILD:37]: Why doesn't "make install" not work?

Comment (pjb7687) [reply mingw-w64-cas-offinder/PKGBUILD:37]: Well I just forgot to implement it in `CMakeLists.txt`. As a developer of `Cas-OFFinder`, I confirm that the compiled binary is the only one you will get after running `make`.

Comment (lazka) [reply mingw-w64-cas-offinder/PKGBUILD:37]: ok

**Pull request merged**